test/unit/query-generator/describe-table-query.test.ts

```diff
@@ -138,43 +131,112 @@ describe('QueryGenerator#describeTableQuery', () => {
           + 'LEFT JOIN sys.extended_properties prop ON prop.major_id = sc.object_id '
           + 'AND prop.minor_id = sc.column_id '
           + `AND prop.name = 'MS_Description' `
-          + `WHERE t.TABLE_NAME = 'myTable'AND t.TABLE_SCHEMA ='mySchema'`,
+          + `WHERE t.TABLE_NAME = N'myTable' AND t.TABLE_SCHEMA = N'mySchema'`,
       sqlite: 'PRAGMA TABLE_INFO(`mySchema.myTable`);',
       db2: 'SELECT NAME AS "Name", TBNAME AS "Table", TBCREATOR AS "Schema", '
           + 'TRIM(COLTYPE) AS "Type", LENGTH AS "Length", SCALE AS "Scale", '
           + 'NULLS AS "IsNull", DEFAULT AS "Default", COLNO AS "Colno", '
           + 'IDENTITY AS "IsIdentity", KEYSEQ AS "KeySeq", REMARKS AS "Comment" '
           + 'FROM SYSIBM.SYSCOLUMNS '
-          + `WHERE TBNAME = 'myTable' AND TBCREATOR ='mySchema';`,
-      ibmi: `SELECT
-          QSYS2.SYSCOLUMNS.*,
-          QSYS2.SYSCST.CONSTRAINT_NAME,
-          QSYS2.SYSCST.CONSTRAINT_TYPE
-          FROM
-          QSYS2.SYSCOLUMNS
-          LEFT OUTER JOIN
-            QSYS2.SYSCSTCOL
-          ON
-            QSYS2.SYSCOLUMNS.TABLE_SCHEMA = QSYS2.SYSCSTCOL.TABLE_SCHEMA
-            AND
-            QSYS2.SYSCOLUMNS.TABLE_NAME = QSYS2.SYSCSTCOL.TABLE_NAME
-            AND
-            QSYS2.SYSCOLUMNS.COLUMN_NAME = QSYS2.SYSCSTCOL.COLUMN_NAME
-          LEFT JOIN
-            QSYS2.SYSCST
-          ON
-            QSYS2.SYSCSTCOL.CONSTRAINT_NAME = QSYS2.SYSCST.CONSTRAINT_NAME
-          WHERE QSYS2.SYSCOLUMNS.TABLE_SCHEMA = 'mySchema' AND QSYS2.SYSCOLUMNS.TABLE_NAME = 'myTable'`,
+          + `WHERE TBNAME = 'myTable' AND TBCREATOR = 'mySchema';`,
+      ibmi: 'SELECT '
+          + 'QSYS2.SYSCOLUMNS.*, '
+          + 'QSYS2.SYSCST.CONSTRAINT_NAME, '
+          + 'QSYS2.SYSCST.CONSTRAINT_TYPE '
+          + 'FROM QSYS2.SYSCOLUMNS '
+          + 'LEFT OUTER JOIN QSYS2.SYSCSTCOL '
+          + 'ON QSYS2.SYSCOLUMNS.TABLE_SCHEMA = QSYS2.SYSCSTCOL.TABLE_SCHEMA '
+          + 'AND QSYS2.SYSCOLUMNS.TABLE_NAME = QSYS2.SYSCSTCOL.TABLE_NAME '
+          + 'AND QSYS2.SYSCOLUMNS.COLUMN_NAME = QSYS2.SYSCSTCOL.COLUMN_NAME '
+          + 'LEFT JOIN QSYS2.SYSCST '
+          + 'ON QSYS2.SYSCSTCOL.CONSTRAINT_NAME = QSYS2.SYSCST.CONSTRAINT_NAME '
+          + `WHERE QSYS2.SYSCOLUMNS.TABLE_SCHEMA = 'mySchema' `
+          + `AND QSYS2.SYSCOLUMNS.TABLE_NAME = 'myTable'`,
+    });
+  });
+
+  it('produces a query to describe a table with default schema in tableName object', () => {
```

Comment:
describeTableQuery should always specify the schema in the query, even if it's the default schema. Test can remain but title should reflect that

Reply:
What would you suggest for the name of the test?

Reply:
I think that if we do the change from the thread below this one, we can just drop this test (even if we do that change in a follow-up PR)

Reply:
I've kept this test since there is a different expectation for `db2` and `ibmi`